### PR TITLE
Update Xunit to build 3109, which has shadow copy fix

### PR DESCRIPTION
**Pull request by tmat**
@KevinH-MS @agocke @tannergooding

### CODE REVIEW

File: src/Debugging/Microsoft.DiaSymReader.PortablePdb.Tests/Microsoft.DiaSymReader.PortablePdb.UnitTests.csproj [hunk 1]
```diff
@@ -120,15 +120,25 @@
...
+      <HintPath>..\..\..\packages\xunit.assert.2.1.0-beta4-build3109\lib\portable-net45+netcore45+wp8+wpa81\xunit.assert.dll</HintPath>
     </Reference>
-    <Reference Include="xunit.core, Version=2.0.0.2785, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
+    <Reference Include="xunit.core, Version=2.0.0.3109, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\packages\xunit.extensibility.core.2.1.0-beta2-build2981\lib\portable-net45+dnxcore50+win+wpa81+wp80+monotouch+monoandroid+Xamarin.iOS\xunit.core.dll</HintPath>
+      <HintPath>..\..\..\packages\xunit.extensibility.core.2.1.0-beta4-build3109\lib\portable-net45+netcore45+wp8+wpa81\xunit.core.dll</HintPath>
     </Reference>
   </ItemGroup>
+  <ItemGroup>
```
Comment (KevinH-MS) [inline src/Debugging/Microsoft.DiaSymReader.PortablePdb.Tests/Microsoft.DiaSymReader.PortablePdb.UnitTests.csproj]: These two ```<Content>``` items shouldn't be needed.  These files just need to exist alongside xunit.core.dll in the packages folder.

**Comment by KevinH-MS** [comment]
@tannergooding this is only changing the projects that already reference xunit 2.0.0 (not the ones referencing xunit 1.9.2).  Should that be ok?

File: src/Debugging/Microsoft.DiaSymReader.PortablePdb.Tests/Microsoft.DiaSymReader.PortablePdb.UnitTests.csproj (hunk 1 above)
Comment (tmat) [inline src/Debugging/Microsoft.DiaSymReader.PortablePdb.Tests/Microsoft.DiaSymReader.PortablePdb.UnitTests.csproj]: You are right. Removed

**Comment by tannergooding** [comment]
@KevinH-MS, if the tests are actually running (and running without issues) then most likely. However, I am hoping to have the other PR finished today which would bring everything up to beta 4.

**Comment by KevinH-MS** [comment]
LGTM :shipit:

**Pull request merged**